### ISSUE TITLE
Keyboard height affects drawer size

### ISSUE THREAD

**Issue by bhandaribhumin** [toniantunovi/ion-bottom-drawer#29]
When I was set select some location and drag drawer to bottom position it will show blank space. 
See the attached 

![4C2CDCDB-F9B3-4087-A4DA-BD8329B0F932](https://user-images.githubusercontent.com/3336047/66015138-0abd3700-e4ef-11e9-8f87-9530e532ae70.png)

![3F708F4F-9997-4998-9372-A045A4A24996](https://user-images.githubusercontent.com/3336047/66015139-0b55cd80-e4ef-11e9-8787-040d964be1e8.png)

Expected behavior 

![F9BA9D34-075B-426E-824D-1FAF4D2E99B6](https://user-images.githubusercontent.com/3336047/66015378-edd53380-e4ef-11e9-9a2d-822bbda11597.png)

**Comment by firebiird** [comment]
I have the same issue, did you resolve?

**Comment by bhandaribhumin** [comment]
yes i fixed using manually set   

```
dockedHeight 
  distanceTop 

after click 

//this code fixed my problem not 100% but i say 98%
 this.dockedHeight = 140;
      setTimeout(() => {
        this.dockedHeight = 140;
        this.drawerState = 2;
      }, 500);
      this.distanceTop = 500;


```

**Comment by Nickholas** [comment]
It doesnt work for me. Any other idea?

**Comment by bhandaribhumin** [comment]
@Nickholas hey what you get after adding this little hackey fix? can you send some video or img so I can take look!

**Comment by Nickholas** [comment]
Yes, of course. I would appreciate your help. Here is an example of my code in GitHub:

https://github.com/Nickholas/demo-bottom-drawler

It works fine on a web client (chrome, firefox,..) but it fails when you try to run it onto an Android device.

![image](https://user-images.githubusercontent.com/4329716/87363400-fe2f3a00-c568-11ea-8f40-eacff43fc776.png)

If you enter a text on the textarea field, it shows the keyboard and then it brokes the height of the component.

![image](https://user-images.githubusercontent.com/4329716/87363333-d8099a00-c568-11ea-9807-9bb8e064a486.png)

Thanks in advanced.
